web/tests/narrow_state.test.js

```diff
@@ -214,7 +216,7 @@ test("set_compose_defaults", () => {
     assert.deepEqual(narrow_state.set_compose_defaults(), {});
 
     stream_data.add_sub({name: "ROME", stream_id: 99});
-    set_filter([["stream", "rome"]]);
+    set_filter([["stream", "99"]]);
```

Comment:
I think we want the pattern for these to look more like `rome_stream.stream_id``, possibly with a `toString` depending on how other comments are resolved. Opaque integers like this make the code hard to read.

Looks like a preexisting issue with this test more broadly, though, two lines down I see a hardcoded 99.

Reply:
Strings for filter terms and numbers for `set_compose_defaults` is the correct typing. But yeah I'm doing a pass right now to not use opaque string integers. (I had a `// TODO(evy): make these more like `channel:${dev_help.stream_id}` below but I wasn't sure how important it was and your comment encouraged me to go ahead and do that)